docs/values.md

```diff
@@ -89,15 +89,15 @@ The `blockedNamespaceList` lists the Kubernetes/Openshift namespaces blocked by
       - kube-system
       - kube-public
 
-### AKOSetttings.istioEnabled (Tech Preview)
+### AKOSetttings.istioEnabled
 
 AKO can be deployed in Istio environment. Setting this to `true` indicates to AKO that the environment is Istio. Default value is `false`.
 
-### AKOSetttings.ipFamily (Tech Preview)
+### AKOSetttings.ipFamily
 
-`V6` is currently supported only for `vCenter` cloud with `calico` CNI.
+`V6` is currently supported only for `vCenter` cloud with `calico` and `antrea` CNI.
```

Comment:
It would be better to add a link to the `support matrix` here.

Reply:
added link references to the support matrix doc